ios/src/platform_ios.mm

```diff
@@ -153,20 +153,23 @@ bool isContinuousRendering() {
 
     for (id fallback in fallbacks) {
 
-        UIFont* font = [UIFont fontWithName:fallback size:1.0];
+        handles.emplace_back([fallback]() {
 
-        size_t dataSize = 0;
-        auto cdata = loadUIFont(font, &dataSize);
+             __block std::vector<char> data;
 
-        if (!cdata) { continue; }
+             dispatch_sync(dispatch_get_main_queue(), ^{
```

Comment:
I realize this is marked [WIP] currently, but just FYI calling dispatch_sync on the main queue in iOS/OS X can quickly lead to a deadlock situation if the call-site calling this function doesn't know that's what this is doing.

You should probably do some defensive programming to check to see if you're already on the main thread, kind of like this answer here: http://stackoverflow.com/a/10341532/490180